src/compile.rs

```diff
@@ -1250,7 +1250,7 @@ test!(basic_function_usage => r#"
 
     export fn main {
       foo();
-      'foo'.bar().print();
+      'foo'.bar.print;
```

Comment:
This is the line that was causing an extra failure before. It was not properly handling method syntax on constants, actually, not the chaining of two sequential method calls without the parens like I originally thought.